blocks/router/router.js

```diff
@@ -0,0 +1,53 @@
+import ComponentBase from '../../scripts/component-base.js';
+
+export default class Router extends ComponentBase {
+  static get observedAttributes() {
+    return ['external'];
+  }
+
+  getPlainUrl(url) {
+    if (url.indexOf('.html') >= 0) {
```

Comment:
typically edge delivery domains do not have an extension, will it still work?

Reply:
then it's the last return
```javascript
return `${url}.plain.html`;
```